editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -154,6 +166,8 @@ function Component(props: Props) {
               if (data) _constraints.push(data as GISResponse["constraints"]);
             }
 
+            const _overrides = inaccurateConstraints;
```

Comment:
Simply writing a new raw `_overrides` key to the passport for now, follow-up PR will work out which values need to be added/removed from `property.constraints.planning` & `_nots` based on override entities and test automations.